sgd/sgd-aa-oas3.yml

```diff
@@ -0,0 +1,577 @@
+openapi: 3.0.3
+info:
+  title: API Sistema Gestione Deleghe (SGD)
+  version: "1.0.0"
+  description: This document contains the specification for the SGD API
+  termsOfService: https://deleghedigitali.gov.it/tos
+  contact:
+    name: Istituto Poligrafico e Zecca dello Stato (IPZS)
+    url: https://deleghedigitali.gov.it
+    email: tech@deleghedigitali.gov.it
+  x-spid:
+    aa-home: https://deleghedigitali.gov.it/api/v1.0.0/
+    aa-registry: https://registry.spid.gov.it/metadata/aa/sgd.json
+    aa-version: 1.0.0
+    aa-required-attributes: [fiscalNumber]
+    aa-lookup-attribute: fiscalNumber
+  x-api-id: 123
+  x-summary: A subtitle for the catalog
+servers:
+  - url: https://deleghedigitali.gov.it/api/v1
+    description: fdsfs
+
+paths:
+  /status:
+    get:
+      operationId: get-status
+      description: fdfd
+      summary: fdfd
+      responses:
+        "200":
+          description: ok
+          content:
+            application/problem+json:
+              schema:
+                type: string
+      tags:
+        - public
+  /versioning:
+    #TODO: Maybe a pagination could be useful for this path
+    get:
+      operationId: get-versioning
+      summary: Returns the list of all available versions of the API
+      description: fdsfsd
+      responses:
+        "200":
+          description: Ok
+          content:
+            text/html:
+              schema:
+                type: string
+            application/json:
+              schema:
+                type: object
+                description:
+                  "JSON Object containing the list of all available versions of
+                  this API."
+                properties:
+                  versions_list:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        iat:
+                          type: string
+                          description:
+                            UNIX timestamp of when this version was released
+                        version:
+                          type: string
+                          description:
+                            Version identifier of API given in the format
+                            'v<MAJOR.MINOR.PATCH>'
+                      required:
+                        - iat
+                        - version
+        "404":
+          $ref: "#/components/responses/404"
+        "500":
+          $ref: "#/components/responses/500"
+      tags:
+        - public
+      x-spid-operation: # See §2.8
```

Comment:
Non lo definirei

Reply:
@peppelinux , se ti riferisci a x-spid-operation, dall'allegato tecnico alle LL.GG. AA risulta quanto segue:
"È prevista per l’oggetto Operation, l’estensione x-spid-operation, con i campi indicati nella
seguente tabella, per descrivere in maniera strutturata le caratteristiche elencate sopra."
Ho interpretato questo come un obbligo, ma forse mi sbaglio.